lib/Map/AddressConverter.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+/*global require*/
+var DeveloperError = require('terriajs-cesium/Source/Core/DeveloperError');
+
+/*
+Determine lat and long coordinates for a given address.
+If TableStructure has address column, adds two new columns: lat and long.
+*/
+
+/**
+ * Instantiate an address converter with a TableStructure that it must convert.
```

Comment:
Anyway this should describe the class, not its constructor.  Something like `Geocodes addresses to a longitude and latitude` maybe?